application/src/main/java/org/togetherjava/tjbot/commands/generic/DatabaseCommand.java

```diff
@@ -31,43 +31,41 @@
  * }
  * </pre>
  */
-public final class DatabaseListener extends ListenerAdapter {
-
-    private static final Logger logger = LoggerFactory.getLogger(DatabaseListener.class);
-
+public class DatabaseCommand extends AbstractCommand {
+    private static final Logger logger = LoggerFactory.getLogger(DatabaseCommand.class);
     private final Database database;
 
     /**
      * Creates a new command listener, using the given database.
      *
      * @param database the database to store the key-value pairs in
      */
-    public DatabaseListener(Database database) {
+    public DatabaseCommand(Database database) {
+        super("Database Get & Set",
```

Comment:
"Database Get & Set" is an invalid command name.
Either name it `database` and make usage of subcommands or create 2 separate commands.

See [this example class ](https://github.com/Together-Java/TJ-Bot/blob/develop/application/src/main/java/org/togetherjava/tjbot/commands/example/SubCommandExample.java#L52) for how to make usage of subcommands

Reply:
Once again, we should have more explicit documentation on that. Going to add it